src/main/java/siri/Storage.java

```diff
@@ -0,0 +1,111 @@
+package siri;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+
+/**
+ * Class that deals with storing and loading of data.
+ */
+class Storage {
+    private File directory;
+    private File dataFile;
+
+    private String fileDirectoryPath;
+    private String filePath;
+
+
+    /**
+     * Constructor of Storage class.
+     *
+     * @param filePath string representation of the file path to allow loading from the file.
+     * If file doesn't exist, file will be created based on the path provided.
+     * If empty String is provided, cwd will create a data folder and a data.txt file inside to be saved.
+     */
+    public Storage(String filePath) {
+
+        if (filePath == "") {
+            this.filePath = "data/data.txt";
```

Comment:
Perhaps you can consider using various path separator variables from File, Files, or use Path imports? 
Helps account for different OS path separators. :)